packages/plugin-lightningcss/tests/index.test.ts

```diff
@@ -0,0 +1,126 @@
+import { expect, describe, it } from 'vitest';
+import { pluginLightningcss, lightningcss } from '../src';
+import { createStubRsbuild } from '@scripts/test-helper';
+
+describe('plugins/lightningcss', () => {
+  it('plugin-lightningcss should replace postcss-loader with lightningcss-loader with default options', async () => {
+    const rsbuild = await createStubRsbuild({
```

Comment:
the `createStubRsbuild` method does not provide a default `postcss-loader`, should we use `createRsbuild` here?